install_config/topics/gce_scopes_permissions.adoc

```diff
@@ -0,0 +1,82 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='configuring-gcp-permissions_{context}']
+= Configuring authorization for Google Cloud Platform
+
+*Roles*
+
+Configuring GCP for {product-title} requires the following GCP role:
+
+[cols='1,3']
+|===
+
+|`roles/owner`
+|Needed for creating service accounts, cloud storage, instances, images,
```

Comment:
s/templates,/templates, and

Reply:
Why is this a stubby table?
And why does this duplicate the same info from the other "concept" topic?